src/components/NextSeoWrapper.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { NextSeo } from 'next-seo';
+
+interface Props {
+  title?: string;
+  description?: string;
+  url?: string;
+  image?: string;
+  imageAlt?: string;
+  imageWidth?: number;
+  imageHeight?: number;
+}
+
+const NextSeoWrapper: React.FC<Props> = (props) => {
+  const { url, image, imageAlt, imageHeight, imageWidth, ...rest } = props;
+  const openGraph = {
+    ...rest,
+    ...(url && { url }),
+    ...(image && {
```

Comment:
I think that you'd want to do something like lodash merge instead, because ...rest only merges the top level children. lmk if it makes sense